opensrs/xcp.py

```diff
@@ -87,18 +87,29 @@ def get_data(self, base_node=None):
         if not ET.iselement(base_node):
             return base_node
         if base_node.tag == 'item':
-            if base_node.getchildren() == []:
-                return base_node.text
+            try:
+                if base_node.getchildren() == []:
```

Comment:
This lib isn't intended to work for python < 2.7. Thus, what if we just replace `.getchildren()` with `list()`?

Reply:
I agree supporting 2.7 isn't ideal, but I wasn't sure what versions are targeted for this library.  I think adding a `python_requires='>=3.x'` and classifiers should be added to clarify the supported versions.  I added another commit to this pull request removing the 2.7 compatibility.

Reply:
I meant that elements are implicitly iterable in Python starting from 2.7 (and `.getchildren()` became deprecated starting from 2.7). Thus we're dropping support of 2.6 and lower here, not 2.7.

Reply:
Thanks for clarifying. Sometimes the docs will say something like "Since Python 3.2 and 2.7"  and I did not realize 2.7 has the same `getchildren()` deprecation.  Either way, I removed it in the latest commit on this pull request.